grails-app/domain/edu/oswego/cs/lakerpolling/domains/User.groovy

```diff
@@ -11,9 +11,9 @@ class User {
     String imageUrl
 
     static hasOne = [authToken: AuthToken, role: Role]
+    static hasMany = [course: Course]
 
     static mapping = {
-        table "users"
```

Comment:
This line is actually necessary. For some reason postgres won't let grails create a table called user.